server/schedule/filter/filters.go

```diff
@@ -30,17 +30,26 @@ import (
 	"go.uber.org/zap"
 )
 
+type StoreFilterResultOption func(*plan.Status, *core.StoreInfo)
```

Comment:
How about making it shorter? BTW, why use the option?

Reply:
> How about making it shorter? BTW, why use the option?
to make collector be an optional parameter in `SelectXXX`